src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -241,6 +241,22 @@ public Instant getInstant(String key) {
   }
 
   /**
+   * Get the value with the specified key, as an generic retrun type
+   * @param key  the key to lookup
+   * @return the value
+   */
+  public <R> R get(String key) {
```

Comment:
@vietj do you see any downside of free/generic class cast? Can you share with me, please. I want to know about this.